MAUI/ImageEditor/shape.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: post
+title: Shape Annotation in .NET MAUI ImageEditor control | Syncfusion
+description: Learn here all about the shape annotation feature of Syncfusion .NET MAUI ImageEditor(SfImageEditor) control.
+platform: .NET MAUI
+control: SfImageEditor
+documentation: ug
+---
+
+# Shape Annotation in .NET MAUI ImageEditor (SfImageEditor)
+
+The image editor control allows you to add various shapes with customizable settings
+
+## Shape types
+
+* Circle
+* Rectangle
+* Arrow
+* Line
+* Dotted
+* DoubleArrow
+* DottedArrow
+* DottedDoubleArrow
+
+## Customize shape settings
+
+The shape annotations can be customized using common annotation customization properties.
+
+* `Bounds`: Specifies the bounds of the annotation view.
+* `AllowDrag`:  Enables or disables the dragging for annotations.
+* `AllowResize`: Enables or disables the resizing for annotations.
+* `Opacity`: Specifies the opacity of annotations.
+
+Shapes can be inserted and customized by changing their stroke color, fill color, and stroke thickness. You can create shapes using either a toolbar or the `AddShape` method.
+
+In the `AddShape` method, you can insert shapes by specifying the fill color, stroke color, and stroke thickness. The [`AddShape`] method has the following parameters.
+
+*  `StrokeThickness` - Specifies the stroke width of the shapes.
+*  `Color` - Specifies the stroke color of the shapes.    
+*  `IsFilled` - Enables or disbles the fill color of the shapes.
+*  `FillColor` - The fill color of the shapes.
+
+{% tabs %}
+{% highlight C# tabtitle="C#" %}
+
+    imageEditor.AddShape(AnnotationShape.Rectangle, new ImageEditorShapeSettings() { Color = Colors.Blue, StrokeThickness = 5, IsFilled =false });
+
+{% endhighlight %}
+{% endtabs %}
+
+## Delete the selected shape
+
+You can delete the selected shape using either the toolbar or the `DeleteAnnotation` method.
+
+{% tabs %}
+{% highlight C# tabtitle="C#" %}
+
+    imageEditor.DeleteAnnotation();
+
+{% endhighlight %}
+{% endtabs %}
+
+## Clear all annotations
+
+You can remove all the annotations using the `ClearAnnotations` method.
+
```

Comment:
Note : It will remove text annotations as well

Reply:
Added a note content